dessia_common/document_generator.py

```diff
@@ -108,6 +108,35 @@ def from_markdown(cls, markdown_text: str):
         return cls(text=markdown_text)
 
 
+class Table:
+    """ Represents a table in the document. """
+
+    def __init__(self, rows: List[List[str]]):
+        self.rows = rows
+
+    def add_to_document(self, document: docx.Document, style="TableGrid"):
+        """
+        Add the table to the document.
+
+        :param document: The document to which the table will be added.
+        :type document: docx.Document
```

Comment:
Useless redundant type documenting (already set in method signature)

Reply:
I didn't completely understand !